docs/tutorial.md

```diff
@@ -234,7 +234,7 @@ The project also contains a number of additional files required for a typical fo
 - `package.json`: Defines your project's Node.js dependencies.
 - `tsconfig.json`: Specifies your TypeScript configuration. Note that Photon.js currently requires the `esModuleInterop` property to be set to `true`.
 - `node_modules/@generated/photon`: Contains the generated Photon.js code. 
-- `script.ts`: Contains the actual "application code", which in this case is a simple script demonstrating some Photon.js API calls.
+- `script.ts`: Contains the actual "application code", which in this case is a script demonstrating some Photon.js API calls.
```

Comment:
Maybe `basic` if we want to highlight that the script is not complicated?

Reply:
Your call. Let me know and I'll update with the other comments!

Reply:
Maybe `sample` can work here better